### DISCUSSION TITLE
WHIP 3: Ambassador Program

### DISCUSSION THREAD

**Discussion by chiumichael** [wihi-labs/WHIP#14] [Ideas]
- Authors: [Mark Ballandies](https://twitter.com/BallandiesMC), [Uroš Kalabić]()
- Start Date: 2023-09-28
- Category: Organization
- Tracking Issue: #4
----
## Summary

The WiHi Ambassador program is a mechanism to grow WiHi’s community of active supporters.  

Link to WHIP:

https://github.com/wihi-labs/WHIP/blob/main/0003-ambassador-program.md

**Comment by urosvk** [comment]
> The community fund is a 3-2-multi-sig wallet that is initially filled with 250000 WiHi tokens

What's the reasoning behind the numbers here?

**Reply by ballandiesmark** [reply]
two out of three need to sign in the multi-sig.
